### DISCUSSION TITLE
Curso Logica de Programacao com Dart (Códigos)

### DISCUSSION THREAD

**Discussion by wendergenz** [balta-io/forum#174] [Tira Dúvidas] [answered]
Ola, não tem mais os códigos prontos das aulas de lógica?
Digito algumas partes dos códigos durante as aulas mas ai o professor tira uns códigos enorme do rascunho dele e fica complicado digitar tudo.
Gostaria de saber se ainda existe alguma repositório com os códigos prontos. 
Desde já agradeço.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @wendergenz, desculpa demora, estava subindo os sources:
https://github.com/balta-io/2816
